elasticdl/python/master/servicer.py

```diff
@@ -65,8 +65,9 @@ def __init__(
         if evaluation_service:
             evaluation_service.set_master_servicer(self)
 
-    # TODO: This is currently being used by multiple tests to initilize
-    # self._model, where the initialization should be done via constructor.
+    # TODO: Multiple tests are currently using function `set_model_var` to
+    # initialize self._model, where the initialization should be done via
+    # constructor.
```

Comment:
I think there are missing `a`s before the word `function` and `constructor`.

Reply:
What is the `contructor`?

Reply:
Done.